tungsten_ci_utils/generate_build_change_info/last_successful.py

```diff
@@ -0,0 +1,95 @@
+import sys
+import argparse
+import MySQLdb
+import logging
+import json
+
+log = logging.getLogger(__name__)
+
+def set_logging():
+    log.setLevel(logging.DEBUG)
+    formatter = logging.Formatter(
+        '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    console = logging.StreamHandler()
+    console.setLevel(logging.DEBUG)
+    console.setFormatter(formatter)
+    log.addHandler(console)
+
+set_logging()
+
+def get_json_data(file):
+    json_file = open(file).read()
```

Comment:
Close the file after reading or use the "with" statement to have it closed automatically:
http://cmdlinetips.com/2016/01/opening-a-file-in-python-using-with-statement/